src/msg/async/dpdk/DPDK.cc

```diff
@@ -532,7 +532,7 @@ bool DPDKQueuePair::init_rx_mbuf_pool()
     for (int i = 0; i < mbufs_per_queue_rx; i++) {
       rte_mbuf* m = rte_pktmbuf_alloc(_pktmbuf_pool_rx);
       assert(m);
```

Comment:
spdk/dpdk changed the refcnt setting check, so need modify it according dpdk library.